docs/source/index.rst

```diff
@@ -120,12 +120,12 @@ Autodetection heuristics
 
 * ``:async-with:`` is autodetected for:
 
-  * functions that have an attribute ``__returns_acontextmanager__`` 
-    (note the ``a``) with a truthy value.
-  
   * functions decorated with `contextlib.asynccontextmanager
     <https://docs.python.org/3/library/contextlib.html#contextlib.asynccontextmanager>`__
 
+  * functions that have an attribute ``__returns_acontextmanager__`` 
```

Comment:
Nit: to fully parallel the above bullets, there should be a comma after the link to `asynccontextmanager`.